.github/workflows/core.yml

```diff
@@ -13,12 +13,12 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         # CPython 3.11 is in quick-test
-        python-version: ['3.8', '3.9', '3.10', '3.12', pypy-3.7, pypy-3.8, pypy-3.10]
+        python-version: ['3.8', '3.9', '3.10', '3.12', pypy-3.8, pypy-3.10]
```

Comment:
Can we add `3.13-dev` here? I would say we should wait until the feature freeze, but we almost forgot to add support for 3.12, so better safe than sorry

Reply:
Tried it.
It's too soon: https://github.com/yt-dlp/yt-dlp/actions/runs/6860253423/job/18653789763?pr=8361

Reply:
```suggestion
        python-version: ['3.9', '3.10', '3.12', pypy-3.8, pypy-3.10]
```
Covered by windows

Reply:
per Grub's comment below we wanted to test oldest and newest py versions on both